lib/attackMonitoring.js

```diff
@@ -170,6 +183,38 @@ let enableAttackMonitoring = function (policyJSON, callbackFunction) {
 
 }
 
+// Send CSP Report.
+function sendReport(reportUri, violationEvent){
+    const data = JSON.stringify(violationEvent);
+    const url = new URL(reportUri);
+    const options = {
+        host: url.hostname,
+        port: url.port,
+        path: url.pathname + url.search,
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/csp-report',
+            'Content-Length': data.length,
+            'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64; rv:101.0) Gecko/20100101 Firefox/101.0',
```

Comment:
Can you please set the User-Agent to 'Node Security Shield/version_number'. 
This will help the CSP monitoring service categorise such alerts easily.